article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -134,7 +134,7 @@ object ArticlePicker {
     val isWhitelisted = whitelist(page.metadata.id)
     val isEnabled = conf.switches.Switches.DotcomRendering.isSwitchedOn
 
-    val tier = if ((isSupported && isWhitelisted && isEnabled && !request.guuiOptOut) || request.isGuui) RemoteRender else LocalRender
+    val tier = if ((isSupported && isWhitelisted && isEnabled) || request.isGuui) RemoteRender else LocalRenderArticle
```

Comment:
I think the confusing thing is that, at this point, we're not really judging whether to remote render AMP, but whether to render AMP at all for this particular content/request.

Reply:
Thanks! Addressing now